### PR TITLE
Metrics scraper

### PR DESCRIPTION
**Pull request by jakub-dzon**
This PR introduces `metrics.HTTPScraper` type that is responsible for calling given HTTP endpoint and scraping (retrieving and parsing) metrics in openmetrics format exposed by it.

### CODE REVIEW

**Review by eloycoto — APPROVED** [review]

**Comment by machacekondra** [comment]
Prometheus SDK is not usable here?

**Review by pkliczewski — APPROVED** [review]

File: internal/metrics/scraper.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	"io"
+	"net/http"
+)
+
+const (
+	acceptHeader = `application/openmetrics-text; version=0.0.1,text/plain;version=0.0.4;q=0.5,*/*;q=0.1`
+)
+
+var (
+	UserAgent = fmt.Sprintf("k4e-device-worker")
```
Comment (masayag) [inline internal/metrics/scraper.go:20]: why not a simple string assignment?

Comment (jakub-dzon) [reply internal/metrics/scraper.go:20]: Good one! Blind copy&pase&edit...

**Comment by jakub-dzon** [comment]
@machacekondra I've only found instrumentation SDK and the closest to what we need was prometheus-commons, which provides openmetrics parser.

**Review by masayag — APPROVED** [review]

**Pull request merged**